src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+
+  if (days >= 7) {
+    return price * days - 50;
+  } else {
```

Comment:
Replace `else` with `else if` to avoid unnecessary nesting.

[More info.](https://www.w3schools.com/js/js_if_else.asp)